### ISSUE TITLE
Live Region: Value in aria-live has no effect 

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/VFO-standards-support#284]
### Summary

Live Region: Value in aria-live has no effect 

1. Save as HTML file:
```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<title>live region</title>
	</head>
	<body>
		<div aria-live=assertive id=1></div>
		<div aria-live=polite id=2></div>
		<div role=alert id=3></div>
		<div role=status id=4></div>
		
		<button onFocus="document.getElementById('1').innerHTML='region1';">live region with aria-live=assertive</button>
		<button onFocus="document.getElementById('2').innerHTML='region2';">live region with aria-live=polite</button>
		<button onFocus="document.getElementById('3').innerHTML='region3';">live region with role=alert</button>
		<button onFocus="document.getElementById('4').innerHTML='region4';">live region with role=status</button>
	</body>
</html> 
```
2. Navigate to all buttons 

### Expected result
* Correct output according to specification (https://www.w3.org/TR/wai-aria-1.2/#aria-live)
	* role=alert and aria-live=assertive with "highest priority and should be presented the user immediately"
	* role=status and aria-live=polite "at the end of speaking the current sentence or when the user pauses typing"
* Consistent output between browsers

### Actual result

* Firefox, IE 11, Chrome: value in aria-live has no effect. The output is always after the other output of the screenreader (i.e. assertive is also output like polite)
* IE 11, Firefox: Only alert is output at the beginning
* Chrome: Also alert is output at the end

See also https://github.com/w3c/aria-practices/issues/78 and many other problems with live regions: https://github.com/FreedomScientific/VFO-standards-support/issues?q=live+region

#### JAWS version and build number

JAWS 2019.1907.42
 
#### Operating System and version

Windows 8

#### Browser and version:

Chrome 76.0.3809.100
Firefox 68.0.2
Internet Explorer 11.0.9600.19431

**Comment by lamador** [comment]
Testing using the latest version of  JAWS (2020.2006.12), Chrome (84.0.4147.89), Firefox (78.0.2), IE11 (11.959.18362.0) on Windows 10 produced the following results:
- JAWS+Chrome: Button name announced before live region for all buttons. Alert role is not announced.
- JAWS+Firefox/IE11: Button name announced before live region except for button with alert role. Role is announced.

By comparison, NVDA (2020.1) announces the live regions first before the button names in all instances. 
- NVDA+Chrome: Live region announced before button name for all buttons. Alert role is not announced.
- NVDA+Firefox: Live region announced before button name for all buttons. Alert role is announced.

In summary, updated made in `aria-live="assertive"` regions are not immediately annouced in JAWS and NVDA as expected and role="alert" is not recognized in Chrome.  Detailed information on live region support can be found on the [Accessibility Support site](https://a11ysupport.io/).
